src/util.cpp

```diff
@@ -890,6 +890,20 @@ bool NetworkUtil::ParseNetworkAddress ( QString       strAddress,
 }
 
 
+CHostAddress NetworkUtil::GetLocalAddress()
+{
+    QTcpSocket socket;
+    socket.connectToHost("8.8.8.8", 53); // google DNS, or something else reliable
+    if (socket.waitForConnected()) {
```

Comment:
In the Qt docu I can read: "QAbstractSocket::waitForConnected(int msecs = 30000)". So we have a default time-out of 30 seconds? If that is the case, this is too long. A connection should not take longer than a few ms. As a Jamulus server user I would expect that at least after 3 seconds the GUI appears, otherwise I would think it is crashed and would kill the process. So I would set the time-out to 3 seconds. Do you agree? Can you please test this time-out that it works correctly by disconncting your laptop/PC from the internet and start the Jamulus server?

Reply:
Ugh, forgot about the server UI - yes, of course.  It'll be easier to test it putting a non-existent IP in.